community/libgit2/build

```diff
@@ -2,10 +2,21 @@
 
 export DESTDIR="$1"
 
-cmake -B build \
+cmake -B build-shared \
```

Comment:
should be build-static because for this one we're not building shared libs

Reply:
fixed